funding/manager.go

```diff
@@ -4112,6 +4128,10 @@ func (f *Manager) newChanAnnouncement(localPubKey,
 	// max_htlc field.
 	msgFlags := lnwire.ChanUpdateRequiredMaxHtlc
 
+	if !shouldAnnounce {
```

Comment:
hmm not sure, so the ourPolicy flag is only not nil, it the channel is a `ZeroConf` channel to make sure we persist the policies which were present before the 6 confirmations. And because the msg flag bits might change see comment below, we cannot use the `ourPolicy` variable ?

```
// We are not reusing the MessageFlags because a channel might
// change from unannounced to announced during its lifetime.
```